compiler/src/iree/compiler/Codegen/Common/ConvertBf16ToUInt16Buffers.cpp

```diff
@@ -209,8 +258,8 @@ struct SPIRVEmulateBf16Pass final
 // Public interface
 //===----------------------------------------------------------------------===//
 
-std::unique_ptr<OperationPass<ModuleOp>> createSPIRVEmulateBf16Pass() {
-  return std::make_unique<SPIRVEmulateBf16Pass>();
+std::unique_ptr<OperationPass<ModuleOp>> createConvertBf16ToUInt16BuffersPass() {
+  return std::make_unique<ConvertBf16ToUInt16BuffersPass>();
```

Comment:
bf16 to uint16 buffers? Why buffers pass?

Reply:
I thought buffers pass because its focused on converting buffer related operations. It could be ` `Bf16BuffersToUInt16Pass` or we can drop the `pass` part of the name. I am not invested heavily in the specific name.